tests/unit/test_client.py

```diff
@@ -1760,6 +1775,24 @@ def test_download_blob_to_file_wo_chunks_w_raw(self):
     def test_download_blob_to_file_w_chunks_w_raw(self):
         self._download_blob_to_file_helper(use_chunks=True, raw_download=True)
 
+    def test_download_blob_have_different_uuid(self):
+        from google.cloud.storage.blob import Blob
+
+        project = "PROJECT"
+        credentials = _make_credentials(project=project)
+        client = self._make_one(credentials=credentials)
+        blob = mock.create_autospec(Blob)
+        blob._encryption_key = None
+        blob._do_download = mock.Mock()
+        file_obj = io.BytesIO()
+        client.download_blob_to_file(blob, file_obj)
+        client.download_blob_to_file(blob, file_obj)
+
+        self.assertNotEqual(
+            blob._do_download.call_args_list[0][0][3]["X-Goog-API-Client"],
+            blob._do_download.call_args_list[1][0][3]["X-Goog-API-Client"],
```

Comment:
just want to confirm the UUID is changing right?

Reply:
Yes, that's what this is testing. 
```
(Pdb) blob._do_download.call_args_list[0][0][3]['X-Goog-API-Client']
'gcloud-python/2.2.1  gl-python/3.8.12 gax/2.7.1 gccl/2.2.1 gccl-invocation-id/90296518-9508-4a41-83f6-247eba7712a2'
(Pdb) blob._do_download.call_args_list[1][0][3]['X-Goog-API-Client']
'gcloud-python/2.2.1  gl-python/3.8.12 gax/2.7.1 gccl/2.2.1 gccl-invocation-id/b7fc4b3a-4cf4-48c1-9475-c95096336c83'
```

Reply:
@Breathtender re-raising this; do you know if there's a separate UUID per resumable upload request? 

I didn't ask this in prior discussions and may have missed it.